_data/chains/eip155-501.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Columbus Test Network",
+  "chain": "CAM",
+  "rpc": ["https://columbus.camino.foundation/ext/bc/C/rpc"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Camino",
+    "symbol": "CAM",
+    "decimals": 18
+  },
+  "infoURL": "https://camino.foundation/",
+  "shortName": "Columbus",
+  "chainId": 501,
+  "networkId": 1001,
+  "icon": "camino",
+  "explorers": [
+    {
+      "name": "blockexplorer",
+      "url": "https://explorer.camino.foundation/columbus",
```

Comment:
this one 404s

Reply:
We're aware of that, we are reworking the pathing in the Block Explorer. If you don't mind, we would like to have this link saved, we will in the coming weeks/days fix it so it will point to the Columbus chain Block Explorer. The reason we used this URL was to not bother you with another PR soon when we change it.

Reply:
I do not want to merge broken links and need to check EIP3091  compatibility